astropy/stats/funcs.py

```diff
@@ -444,6 +446,196 @@ def true_efficiency(x):
     return bin_ctr, bin_halfwidth, p, perr
 
 
+def poisson_conf_interval(n, interval='root-n', sigma=1):
+    r"""Poisson parameter confidence interval given observed counts
+    
+    Parameters
+    ----------
+    n : int or numpy.ndarray
+        Number of counts (0 <= ``n``).
+    interval : {'root-n','root-n-0','pearson','sherpagehrels','frequentist-confidence'}, optional
+        Formula used for confidence interval. See notes for details.
+        Default is ``'root-n'``.
+    sigma : float
+        Number of sigma for confidence interval; only supported for
+        the 'frequentist-confidence' mode.
+    
+
+    Returns
+    -------
+    conf_interval : numpy.ndarray
+        ``conf_interval[0]`` and ``conf_interval[1]`` correspond to the lower
+        and upper limits, respectively, for each element in ``n``.
+
+    Notes
+    -----
+
+    The "right" confidence interval to use for Poisson data is a
+    matter of debate. The CDF working group [recommends][pois_eb]
+    using root-n throughout, largely in the interest of
+    comprehensibility, but discusses other possibilities. The ATLAS
+    group also [discusses][ErrorBars] several possibilities but
+    concludes that no single representation is suitable for all cases.
+    The suggestion has also been [floated][ac12] that error bars should be
+    attached to theoretical predictions instead of observed data,
+    which this function will not help with (but it's easy; then you
+    really should use the square root of the theoretical prediction).
+
+    The intervals implemented here are:
+
+    **1. 'root-n'** This is a very widely used standard rule derived
+    from the maximum-likelihood estimator for the mean of the Poisson
+    process. While it produces questionable results for small n and
+    outright wrong results for n=0, it is standard enough that people are
+    (supposedly) used to interpreting these wonky values. The interval is
+
+    .. math::
+
+        CI = (n-\sqrt{n}, n+\sqrt{n})
+
+    **2. 'root-n-0'** This is identical to the above except that where
+    n is zero the interval returned is (0,1).
+        
+    **3. 'pearson'** This is an only-slightly-more-complicated rule
+    based on Pearson's chi-squared rule. It also has the nice feature that
+    if your theory curve touches an endpoint of the interval, then your
+    data point is indeed one sigma away. The interval is
+
+    .. math::
+
+        CI = (n+0.5-\sqrt{n+0.25}, n+0.5+\sqrt{n+0.25})
+
+    **4. 'sherpagehrels'** This rule is used by default in the fitting
+    package 'sherpa'. The documentation claims it is based on a numerical
```

Comment:
Instead of pointing out a small error in the Sherpa docs from the Astropy docs, the statement in the Sherpa docs should be corrected. If the incorrect statement is in the code it can be fixed via a PR here: https://github.com/sherpa/sherpa If it is in some other parts of their docs or on the website, a ticket can be files here http://cxc.harvard.edu/helpdesk/ . On both the response / fix usually happens within a few days.